asciidoc/getting_started/azure-react/plugin-information.adoc

```diff
@@ -0,0 +1,39 @@
+== Configures a ReactJS application for Azure!
+
+Configures your ReactJS application to use various Azure services:
+
+- https://www.npmjs.com/package/applicationinsights[App Insights]
+- ...more to come.
+
+=== Setting up @ensono-stacks/azure-react
+
+Install the `@ensono-stacks/azure-react` plugin with the following command:
+
+[source, bash]
+npm install --save-dev @ensono-stacks/azure-react@latest
+
+or
+
+[source, bash]
+yarn add --dev @ensono-stacks/azure-react@latest
+
+=== Executors and Generators
+
+To see a list of the plugin capabilities run the following command:
+
+[source, bash]
+nx list @ensono-stacks/azure-react
+
+View additional information about a plugin capability through the following commands:
+
+[source, bash]
+nx g @ensono-stacks/azure-react:app-insights-web --help
+
+=== Generators
+
+.Installs and configures a ReactJS Library with App Insights.
+[%collapsible]
+====
+include::generator-app-insights-web.adoc[]
```

Comment:
Do we need this one to be collapsible, as its only the link being hidden?

Reply:
true, removed